### PR TITLE
README.md: Some grammatical fixes are made.

### PR DESCRIPTION
**Pull request by varikvalefor**

### CODE REVIEW

**Comment by MrMasrozYTLIVE** [comment]
Uhh bro you Just added some "." at end of some Word, edit Chrome from small letter to big and smthing other not important. Lol

**Comment by varikvalefor** [comment]
> Uhh bro you Just added some "." at end of some Word, edit Chrome from small letter to big and smthing other not important. Lol

Blessed are the pedants.

File: README.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ delete debugMenu.customEntries['myKey']
 
 ### Some debugging examples
```
Comment (AwesomestCode) [inline README.md:93]: Not something you necessarily need to change; but as a note to myself and other maintainers, it's probably worth deciding if we want to capitalise section titles or not.

**Review by AwesomestCode — CHANGES_REQUESTED** [review]
Looks good, just a few minor issues. Thank you very much for the contributions!

On a side note: I noticed that there were some more punctuation errors in other parts of the readme (like spaces before punctuation marks). It's totally fine if you don't have the time or will to correct them, but it would very helpful if you did. If you don't want to, I'll do it myself later.

File: README.md [hunk 2]
```diff
@@ -70,13 +70,13 @@ and you can refresh the page to get the new result.
...
+You may want to disable auto saving in your IDE to avoid constant rebuilding; see https://webpack.js.org/guides/development/#adjusting-your-text-editor.
 
-To check the production build (take a minute to build), you can run `npm run build-start`
+To check the production build (take a minute to build), you can run `npm run build-start`.
 
-If you're interested in contributing, you can check projects at https://github.com/PrismarineJS/prismarine-web-client/projects
+If you're interested in contributing, you can check projects at https://github.com/PrismarineJS/prismarine-web-client/projects.
 
-Some variables are exposed in window for debugging:
+Some variables are exposed in the window for debugging:
```
[the remark this answers is not in the document]
Comment (varikvalefor) [reply README.md:79]: Using the <CODE /> equivalent is probably the best choice.

**Comment by varikvalefor** [comment]
> Thank you very much for the contributions!

De nada.

> On a side note: I noticed that there were some more punctuation errors in other parts of the readme (like spaces before certain punctuation marks). It's totally fine if you don't have the time or will to correct them, but it would very helpful if you did.

These problems shall be investigated.

**Comment by varikvalefor** [comment]
@AwesomestCode: The problems are investigated and fixed.  Should a new pull request be made?  GitHub is weird.

**Review by AwesomestCode — APPROVED** [review]
Made some minor edits myself to fix a couple of small things. LGTM.
Once again, thank you for contributing!

**Pull request merged**